index.html

```diff
@@ -154,35 +160,38 @@
     <div id="sidebar"
       class="sidebar">
       <div class="flex flex-col overflow-y-auto w-full bg-white">
-        <div class="space-y-6 p-5 text-slate-700 z-20 bg-white">
+        <div class="space-y-6 px-12 py-10 text-slate-700 z-20 bg-white">
           <div
-            class="flex items-center min-h-5 h-5 pl-5 border-l-4 border-teal-500">
+            class="flex font-serif items-center min-h-5 h-5 pl-5 border-l-4 border-mod-sc-ev">
             <h2 class="leading-[18px] text-lg"><span
                 class="text-slate-700">Explore
                 the </span><span
-                class="text-teal-500 font-semibold">impact
+                class="text-mod-sc-ev font-semibold">impact
                 of different interventions</span><span
                 class="text-slate-700">
                 on soil organic carbon.</span></h2>
           </div>
-          <!-- LAND USE MENU -->
-          <!-- Loaded dynamically on dynamic-data.js -->
-          <div class="flex gap-2 pt-6"
-            id="land-use-menu"></div>
           <div
             id="land-use-intro"
-            class="font-serif leading-10 text-2xl"><span>These
+            class="font-serif leading-10 text-2xl mt-10">
+            <span>These
               insights come from the analysis of</span><span
               class="font-semibold">
               <span id="land-use-publications"></span> scientific publications</span><span>
               and <span id="land-use-meta-analysis"></span> meta-analysis
               related
               to </span><span
               class="font-semibold" id="land-use-text"></span><span>
-              interventions on SOC.</span>
+              interventions on SOC.
+            </span>
           </div>
+
+          <!-- LAND USE MENU -->
+          <!-- Loaded dynamically on dynamic-data.js -->
+          <div class="flex gap-2 pt-6"
+            id="land-use-menu"></div>
```

Comment:
There's already a 24px gap, no need for a padding here:
```suggestion
          <div class="flex gap-2"
            id="land-use-menu"></div>
```